wrappers/mmapwrapper.h

```diff
@@ -174,6 +181,16 @@ namespace HL {
 	perror("MmapWrapper");
 	return nullptr;
       } else {
+#if HL_NAMED_HEAP
+        // For every anonymous map (assuming CONFIG_ANON_VMA_NAME is on), the range appears in /proc/<pid>/maps as
+	// `560215e4f000-560215e81000 rw-p 00000000 00:00 0 7ffaf89c0000-7ffaf8aa1000 rw-p 00000000 00:00 0 [anon:Heap Layer]`
+	//
```

Comment:
Fix comment to reflect "Heap Layers" and also say it can be set by defining `HL_HEAP_NAME` (and point to `heaplayers.h` for default definition).